### PR TITLE
add gedit integration fix

### PR DESCRIPTION
**Pull request by Architrixs**
Fixed the variable thing.
and..
Screenshot from 2021-02-04 11-21-28

### CODE REVIEW

**Comment by ichard26** [comment]
pre-commit is still failing? also, I wanna give this a review soon.

File: docs/editor_integration.md [hunk 1]
```diff
@@ -233,6 +233,33 @@ If you later want to update _Black_, you should do it like this:
...
+```
+
+- Set a keyboard shortcut if you like, Ex. `ctrl-B`
+- Save: `Nothing`
+- Input: `Nothing`
+- Output: `Display in bottom pane` if you like.
+- Change the name of the tool if you like.
+
+Use your keyboard shortcut or `Tools -> External Tools` to use your new tool.
+When you close and Reopen your File, black will be done with its job.
```
Comment (ichard26) [inline docs/editor_integration.md:261]: stylistic nits
```suggestion
When you close and reopen your File, _Black_ will be done with its job.
```

**Review by ichard26 — CHANGES_REQUESTED** [review]
The bash script works just fine now, thanks!

Please address the failing prettier pre-commit hook once again and the style nits and this should be ready to go! Thanks again, if you want to, go add yourself to the AUTHORS list in the readme, you deserve it.

**Comment by Architrixs** [comment]
pre-commit keeps failing here (even tho it passes on my ubuntu locally), so I'm gonna delete this branch and try from start and cloning again.

**Pull request closed without merging**